docs/rancher/index.md

```diff
@@ -0,0 +1,192 @@
+# Rancher Cookbook
+
+Recipe for installing
+_Sextant | Community_ edition on a Rancher managed kubernetes cluster to deploy
+and manage blockchain networks.
+
+## Install Prerequisite Tools
+
+You will need the up-to-date versions of the following tools installed -
+
+* [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl)
+
+## Sign up for an evaluation licence
+
+To install _Sextant | Community_ edition, you will need credentials
```

Comment:
Can we lose italics and go with Sextant Community Edition here.

Reply:
Done

Reply:
should be correct now